src/Sylius/Behat/Context/Setup/CurrencyContext.php

```diff
@@ -122,6 +117,48 @@ public function theStoreHasCurrencyWithExchangeRate($currencyCode, $exchangeRate
     }
 
     /**
+     * @Given /^(that channel) allows to shop using the "([^"]+)" currency$/
+     * @Given /^(that channel) allows to shop using "([^"]+)" and "([^"]+)" currencies$/
+     * @Given /^(that channel) allows to shop using "([^"]+)", "([^"]+)" and "([^"]+)" currencies$/
+     */
+    public function thatChannelAllowsToShopUsingAndCurrencies(
```

Comment:
But it has this nasty foreach. But it just a small semantic change. I would prefer to have them separated, but this is irrelevant.